test/api/find-or-create.test.js

```diff
@@ -81,7 +121,12 @@ test('double findOrCreate should not create two v1 feeds', async (t) => {
   const shardAnnouncements = msgs.filter((msg) =>
     hexes.includes(msg.value.content.feedpurpose)
   )
-  t.equals(shardAnnouncements.length, 1, 'only one shard announcement')
+  // sometimes :shardA and :shardB are the same, among
+  // root/v1/:shardA/main and root/v1/:shardB/chess
```

Comment:
:hankey: 

Ideally we have a way to deterministically create the root mf keys so we get a good test for things like this.
I'm happy know this would intermitently fail if there was something genuinely wrong, so this seems like a fine solution

Reply:
Yeah, I also feel poopy about this. We should definitely add a seed for the root, for tests, but it's a bit more elaborate than this PR can fit. Should/could be done soon.